src/azul/project/hca/__init__.py

```diff
@@ -6,12 +6,17 @@
 
 from azul.indexer import BaseIndexer
 import azul.plugin
+from azul.portal_service import PortalService
 from azul.project.hca.indexer import Indexer
 from azul.types import JSON
 
 
 class Plugin(azul.plugin.Plugin):
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._staged_portal_integrations_db = PortalService.stage_db(Plugin._unstaged_portal_integrations_db())
```

Comment:
It seems a bit excessive to be creating the "staged" portal db every time the Plugin class is loaded, especially since we only expect this hardcoded db to be used once before the db is saved to s3.

Instead of pre-setting up the default db so it can be passed into `get_portal_integrations_db()`, what if the default db was created in `get_portal_integrations_db()` (by calling `PortalService.stage_db()`) when it determined the db on s3 didn't exist?

Reply:
I would propose that we instead keep the design of `PoralService` the same and change `azul.project.hca.Plugin._staged_portal_integrations_db` to a class attribute that is only initialized once.

Reply:
Ok, yes that sounds better.